### PR TITLE
add rightonly jsondiff syntax

### PR DESCRIPTION
**Pull request by ramwin**
I some senario, we often compare old value and new value and check if the newer value is reasonabel. So the rightonly syntax will compute the dicfference and keep the later one compared to CompactJsonDiffSyntax

### CODE REVIEW

File: jsondiff/symbols.py [hunk 1]
```diff
@@ -12,6 +12,10 @@ def __str__(self):
     def __eq__(self, other):
         return self.label == other.label
 
+    def __hash__(self):
```
[the remark this answers is not in the document]
Comment (fzumstein) [reply jsondiff/symbols.py:15]: this was fixed in #61

**Review by fzumstein — COMMENTED** [review]
didn't know you have to submit review for comments to become visible. anyhow, the hash issue has now been fixed.

File: tests/test_jsondiff.py [hunk 2]
```diff
@@ -8,6 +9,15 @@
 from hypothesis import given, settings, strategies
 
 
+logging.basicConfig(
```
Comment (corytodd) [inline tests/test_jsondiff.py:12]: This seems like debugging code. Did you intend to leave this in here?

**Comment by corytodd** [comment]
IntuitiveSyntax is interesting! If you're still working on this it would be really cool to have as an option. Maybe a followup commit to strip this PR down to just that feature?

**Comment by corytodd** [comment]
After testing this some more I'm questioning the name `intuitive`. The names of the other syntax objectively describe their output so I worry that others might not find this output intuitive for their data.

Some ideas:
- unified:  output has full context which reminds me of `git diff --unified`
- compact-context:  compact is extended to include more context

File: tests/test_jsondiff.py (hunk 2 above)
Comment (ramwin) [reply tests/test_jsondiff.py:12]: It's convenient to add some log during test.

File: jsondiff/symbols.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramwin) [reply jsondiff/symbols.py:15]: ok

**Comment by ramwin** [comment]
In my senario, I only what to know what field has changed. And check if the newest value is reasonable.  
How about using 'rightonly', 'keepright' as the syntax name?

**Comment by corytodd** [comment]
Those sound reasonable to me! +1 for `rightonly`.

**Comment by corytodd** [comment]
Can you edit the title to reflect the latest changes and add a PR description explaining your changes? That will make it easier to understand months from now.

**Comment by ramwin** [comment]
@corytodd I have updated the pull request title and added a PR description. Thank you for your advice.

**Review by corytodd — APPROVED** [review]
LGTM

**Comment by fzumstein** [comment]
thanks @ramwin and @corytodd !

**Comment by bew** [comment]
Hello, do you intend to make a release soon with this PR?

**Pull request merged**